unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/SceneAB/AssetPromise_SceneAB.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL;
+using DCL.Helpers;
+using System;
+using System.Threading;
+using UnityEngine;
+
+namespace MainScripts.DCL.Controllers.AssetManager.AssetBundles.SceneAB
+{
+    [Serializable]
+    public class SceneAbDto
+    {
+        public string version;
+        public string[] files;
+        public int exitCode;
+    }
+
+    public class AssetPromise_SceneAB : AssetPromise_WithUrl<Asset_SceneAB>
+    {
+        private readonly CancellationTokenSource cancellationTokenSource;
+        private Service<IWebRequestController> webRequestController;
+
+        private Action onSuccess;
+        private Action<Exception> onFail;
+
+        public AssetPromise_SceneAB(string contentUrl, string hash) : base(contentUrl, hash)
+        {
+            cancellationTokenSource = new CancellationTokenSource();
+        }
+
+        protected override void OnCancelLoading()
+        {
+            cancellationTokenSource.Cancel();
+            cancellationTokenSource.Dispose();
+        }
+
+        protected override void OnLoad(Action OnSuccess, Action<Exception> OnFail)
+        {
+            onFail = OnFail;
+            onSuccess = OnSuccess;
+
+            var kernelConfigPromise = KernelConfig.i.EnsureConfigInitialized();
+            kernelConfigPromise.Catch(error => onFail(new Exception(error)));
+            kernelConfigPromise.Then(k => AsyncOnLoad(k).Forget());
+        }
+
+        private async UniTaskVoid AsyncOnLoad(KernelConfigModel kernelConfigModel)
+        {
+            var tld = kernelConfigModel.network == "mainnet" ? ".org/" : ".zone/";
```

Comment:
This _must_ be part of the scene, we are currently setting `baseUrlAssetBundles`, this could be `baseUrlAssetBundlesCdn`.

It may use different CDNs for worlds and wearables, both valid at the same time